services/movie.py

```diff
@@ -0,0 +1,34 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: None = None,
+        actors_ids: None = None
+) -> None:
+    queryset = Movie.objects.all()
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids).distinct()
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids).distinct()
+    return queryset
+
+
+def get_movie_by_id(movie_id: id) -> id:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: None,
+        movie_description: None,
+        genres_ids: None = None,
+        actors_ids: None = None
+) -> None:

```

Comment:
The function `create_movie` should not return `None`. It should return the created `Movie` instance. Update the return type to `Movie`.